src/sql/workbench/services/query/common/queryModelService.ts

```diff
@@ -170,7 +172,8 @@ export class QueryModelService implements IQueryModelService {
 
 	public async copyResults(uri: string, selection: Slick.Range[], batchId: number, resultId: number, includeHeaders?: boolean): Promise<void> {
 		if (this._queryInfoMap.has(uri)) {
-			return this._queryInfoMap.get(uri)!.queryRunner!.copyResults(selection, batchId, resultId, includeHeaders);
```

Comment:
@alanrenmsft I think you missed updating this call here with your copy result refactoring, `includeHeaders` is actually being passed in as `shouldRemoveNewLines` in runnner